backend/forecastbox/products/plots/maps.py

```diff
@@ -117,9 +118,16 @@ def quickplot(
     from earthkit.plots.schemas import schema
     from earthkit.plots.utils import iter_utils
 
-    schema.use(config.product.plots_schema)
 
-    schema.use_preferred_units = True
+    selected_schema = config.product.plots_schema
+    schema_dir = (Path(__file__).parent / "schemas").resolve().absolute()
```

Comment:
while this work, it is more robust to use https://docs.python.org/3.11/library/importlib.resources.html#module-importlib.resources here

something like
```
schema_dir = importlib.resources.files(forecastbox.products.schemas)
```
that gives you a PosixPath regardless of whether you are in editable, wheel, zip, ... install of the package, possibly missing some methods if its like a temporary unzipped file etc. Not sure why that `inbuilt` workaround is needed below, but the PosixPath object has eg `read_text` method which you can work with universally